blueprints/ember-decorators/index.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  normalizeEntityName: function() {}, // no-op since we're just adding dependencies
+
+  afterInstall: function() {
+    return this.addPackagesToProject([
+      { name: 'babel-plugin-transform-class-properties' },
```

Comment:
We shouldn't need to add these to the project, instead we add them as our own deps and just require them in the babel config (instead of providing a string)